sitka-insights.php

```diff
@@ -55,15 +51,22 @@
   return $client ?: new Client([
     'key'        => get_option('sitka_api_key'),
     'collection' => get_option('sitka_collection_id'),
-    'baseUri'    => get_option('sitka_base_uri'),
+    'baseUri'    => apply_filters('sitka/api/base_uri', ''),
   ]);
 });
 
+add_filter('sitka/api/base_uri', function() {
+  $env  = get_option('sitka_environment');
+  $uris = [
+    'production' => 'https://prd.search-api-gateway.aws.gearlabnw.net',
+    'staging'    => 'https://stg.search-api-gateway.aws.gearlabnw.net',
+  ];
 
-/*
- * Disable default search
- */
-add_action('init', Sitka\disable_default_wp_search::class);
+  $uri  = $uris[$env] ?? $uris['production'];
```

Comment:
It looks like this plugin requires at least PHP 7.0. It might be worth using the "Requires PHP" [header field](https://developer.wordpress.org/plugins/plugin-basics/header-requirements/#header-fields) in your comment at the top of this file. It's unlikely we'd run into issue with this on our end, but clients and/or outside devs could run into an issue trying to install the plugin on a server running an older PHP version.

Reply:
Good call! Added.